[Docs] Add environment variable options to deployment docs (#809)
mckinsey/vizro: vizro-core/docs/pages/user-guides/run.md

[`Vizro`][vizro.Vizro] accepts `**kwargs` that are passed through to `Dash`. This enables you to configure the underlying Dash app using the same [arguments that are available](https://dash.plotly.com/reference#dash.dash) in `Dash`. For example, in a deployment context, some of the below arguments may be useful:

> - `url_base_pathname`: serve your Vizro app at a specific path rather than at the domain root. For example, if you host your dashboard at http://www.example.com/my_dashboard/ then you would set `url_base_pathname="/my_dashboard/"`. Alternatively, you can set the environment variable `DASH_URL_BASE_PATHNAME`.

Reviewer: I know Vale will suggest using "or" rather than "alternatively". Maybe we could say, "If you prefer, you can set the environment variable..." (also, how do you set it? Could you include the full text, as I assume it needs a path. Is it `DASH_URL_BASE_PATHNAME ="/my_dashboard/"` or ??)

> - `url_base_pathname`: serve your Vizro app at a specific path rather than at the domain root. For example, if you host your dashboard at http://www.example.com/my_dashboard/ then you would set `url_base_pathname="/my_dashboard/"`. Alternatively, you can set the environment variable `DASH_URL_BASE_PATHNAME`.

Author: ```suggestion
- `url_base_pathname`: serve your Vizro app at a specific path rather than at the domain root. For example, if you host your dashboard at http://www.example.com/my_dashboard/ then you would set `url_base_pathname="/my_dashboard/"`. You could also set this using an environment variable `DASH_URL_BASE_PATHNAME="/my_dashboard"/`.
```

- `serve_locally`: set to `False` to [serve Dash component libraries from a Content Delivery Network (CDN)](https://dash.plotly.com/external-resources#serving-dash's-component-libraries-locally-or-from-a-cdn), which reduces load on the server and can improve performance. Vizro uses [jsDeliver](https://www.jsdelivr.com/) as a CDN for CSS and JavaScript sources.

> - `assets_external_path`: when `serve_locally=False`, you can also [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn).  Alternatively, you can set the environment variable `DASH_ASSETS_EXTERNAL_PATH`.

Author: ```suggestion
- `assets_external_path`: when `serve_locally=False`, you can also set `assets_external_path` to [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn).  Alternatively, you can set the environment variable `DASH_ASSETS_EXTERNAL_PATH`.
```

> - `assets_external_path`: when `serve_locally=False`, you can also set `assets_external_path` to [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn). Alternatively, you can set the environment variable `DASH_ASSETS_EXTERNAL_PATH`.

Author: ```suggestion
- `assets_external_path`: when `serve_locally=False`, you can also set `assets_external_path` or an environment variable `DASH_ASSETS_EXTERNAL_PATH` to [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn).
```

> - `assets_external_path`: when `serve_locally=False`, you can also set `assets_external_path` to [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn).  Alternatively, you can set the environment variable `DASH_ASSETS_EXTERNAL_PATH`.

Author: ```suggestion
- `assets_external_path`: when `serve_locally=False`, you can also set `assets_external_path` to [serve your own assets from a CDN](https://dash.plotly.com/external-resources#load-assets-from-a-folder-hosted-on-a-cdn). Alternatively, you can set the environment variable `DASH_ASSETS_EXTERNAL_PATH`.
```
